orbit-2-core/README.md

```diff
@@ -0,0 +1,319 @@
+# Orbit 2 Core
+
+This is the core module for the Orbit 2 framework. It's worth familiarising
+yourself with the [overview documentation](docs/orbit-2-overview.md) before
+reading this section.
+
+The core module provides all the basic parts of Orbit:
+
+- Public API
+- Core DSL
+- Plugin registry
+- Container implementation
+
+## Architecture
+
+![Orbit 2 overview 1](docs/orbit-2-overview-1.svg)
+
+This diagram shows a simple representation of how an Orbit system (or similar
+systems like MVI/Redux/Cycle) works in simple principles.
+
+1. The UI sends actions asynchronously to some business component
+1. The business component applies some business logic to the incoming events
+1. The business component then emits some events further down the chain
```

Comment:
would probably avoid the use of the word "some" throughout this... plus would it help to use some of the language of thee library here too?

```
1. The UI sends actions asynchronously to a business component
1. The business component transforms the incoming events with business logic
1. The business component then emits these events further down the chain
```